src/home/components/Skills/styles.module.scss

```diff
@@ -0,0 +1,99 @@
+@import '/src/styles/theme.scss';
+@import '/src/styles/media_queries.scss';
+
+.title {
+  margin: 100px 0px 100px 0px;
+  letter-spacing: 1px;
+  line-height: 60px;
+  text-align: center;
+}
+
+.listSkill {
+  width: 100%;
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
```

Comment:
Set the `grid-template-columns` property to `repeat(auto-fill, minmax(200px, auto))` so you don't have to set the same property within tablet and mobile styles, as browser will render the appropriate number of columns depending on the screen width.